projects/00-trivia/src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="es" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
```

Comment:
Por temas de accesibilidad se considera mala práctica `maximum-scale=1.0`.

```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Ver https://a11yproject.com/posts/never-use-maximum-scale/